README.md

```diff
@@ -153,6 +153,56 @@ acceptance_models = acceptance_model_creator.create_acceptance_map_per_observati
                                                                                    zenith_interpolation=True)
 ```
 
+## Using off runs for background model
+
+It's also possible to create a model from OFF runs and to apply in on runs you want to analyse. The exclusions regions should cover potential sources both in the on and off runs. The OFF runs doesn't need to be connected.
+```python
+acceptance_model_creator = RadialAcceptanceMapCreator(energy_axis_acceptance,
+                                                      offset_axis_acceptance,
+                                                      exclude_regions=exclude_regions,
+                                                      oversample_map=10,
+                                                      min_run_per_cos_zenith_bin=3,
+                                                      initial_cos_zenith_binning=0.01)
+acceptance_models = acceptance_model_creator.create_acceptance_map_per_observation(obs_collection,
+                                                                                   off_observations=obs_collection_off,
+                                                                                   zenith_binning=True,
+                                                                                   zenith_interpolation=True)
+```
+
+## Store background model for later application
+
+It could be in some case usefull to precompute a model and applying it on data later. The example below cover the case where you want to use a model per run using zenith interpolation and off runs. But it's possible to use this functionality without off runs or zenith interpolation. In the last case you just need to provide a model in a gammapy format.
```

Comment:
The main point of having access to intermediary results is storage in my opinion (even if it could also be use for investigation). I've updated a bit the text to emphase than the storage is simply a use of pickle.